.github/workflows/push-tag-create-release.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         with:
           name: release-metadata
           path: release-metadata.json
-          retention-days: 3
+          retention-days: 30
```

Comment:
What's the motivation here?

Reply:
I had some trouble debugging since all the artifacts were gone. Now that we're releasing ~monthly, this felt like a reasonable time period to ensure there would usually be an artifact around to use for investigation/debugging. The file is also super small. Regardless, though, debugging was my main reason. Probably worth a comment.